### PR TITLE
0069-feel-list

### PR DESCRIPTION
**Pull request by StrayAlien**

### CODE REVIEW

File: TestCases/compliance-level-3/0069-feel-list/0069-feel-list-test-01.xml [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+        <description>index of 0 on non list - boolean</description>
+        <resultNode name="decision021" type="decision">
+            <expected>
+                <value xsi:nil="true"/>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="022">
+        <description>index of 01 on non list - number</description>
```
Comment (tarilabs) [inline TestCases/compliance-level-3/0069-feel-list/0069-feel-list-test-01.xml]: typo I believe here is meant `index of 0`.

File: TestCases/compliance-level-3/0069-feel-list/0069-feel-list-test-01.xml [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+        <resultNode name="decision022" type="decision">
+            <expected>
+                <value xsi:nil="true"/>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="023">
+        <description>index of 1 on non list - string</description>
```
Comment (tarilabs) [inline TestCases/compliance-level-3/0069-feel-list/0069-feel-list-test-01.xml]: typo I believe here is meant `index of 0`.

**Comment by agilepro** [comment]
Please address typos and then we will merge

**Comment by StrayAlien** [comment]
@agilepro typos fixed.

**Comment by opatrascoiu** [comment]
What is the rationale behind test `list_011`

         `[1,2,3] in [1,2,3]`

having an expected result `false`? I believe the semantics of the `in` operator is described in Table 52.

**Comment by opatrascoiu** [comment]
Looks fine to me.

**Pull request merged**